### PR TITLE
making namespace usage more consistent

### PR DESCRIPTION
**Pull request by christopherhadley**
- [x] closes #37838
- [x] tests added / passed
- [x] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/arithmetic/test_numeric.py [hunk 1]
```diff
@@ -1054,7 +1054,7 @@ def test_ufunc_compat(self, holder):
         tm.assert_equal(result, expected)
 
     @pytest.mark.parametrize(
-        "holder", [pd.Int64Index, pd.UInt64Index, pd.Float64Index, Series]
+        "holder", [Int64Index, pd.UInt64Index, pd.Float64Index, Series]
```
Comment (jbrockmendel) [inline pandas/tests/arithmetic/test_numeric.py:1057]: lets add these other Index classes at the top while we're here

**Comment by jbrockmendel** [comment]
small comment, ping on green

File: pandas/tests/arithmetic/test_numeric.py (hunk 1 above)
Comment (christopherhadley) [reply pandas/tests/arithmetic/test_numeric.py]: Thanks for the suggestion - I've added these too.

**Comment by christopherhadley** [comment]
I've made the suggested changes.

**Comment by jbrockmendel** [comment]
thanks @christopherhadley

**Pull request merged**